### PR TITLE
feat: add explanatory banner for cascading end times auction

### PR DESCRIPTION
**Pull request by mzikherman**
Looks like:

Mar-16-2022 13-46-13

Updated screenshot (w/o link):
Screen Shot 2022-03-21 at 11 36 02 AM

When the link to the help center article is available, we can set it via ENV (to avoid needing code/deploy).

### CODE REVIEW

File: src/v2/Apps/Auction/AuctionApp.tsx [hunk 1]
```diff
@@ -38,6 +39,9 @@ export const AuctionApp: React.FC<AuctionAppProps> = ({
     showBuyNowTab: sale.showBuyNowTab,
   }
 
+  const { cascadingEndTimeInterval } = sale
+  const isFullBleedHeaderFixed = !cascadingEndTimeInterval
```
Comment (mzikherman) [inline src/v2/Apps/Auction/AuctionApp.tsx:43]: Brought this up as a possible design question in Slack, but when we display this banner we can't have the full bleed header image have a fixed position.

**Review by nspinazz89 — DISMISSED** [review]
LGTM, a spec would be a nice to have but not completely necessary as this seems pretty straight forward.

**Comment by mzikherman** [comment]
Totally right @nspinazz89 ! Added a spec.

**Comment by mzikherman** [comment]
Screen Shot 2022-03-17 at 4 32 49 PM

Updated to look like this with the current design. We can still configure the 'Learn More' link when it's ready (it will not be shown unless set), so I believe this is mergeable before then.

**Review by annacarey — DISMISSED** [review]
Looks good! I'll take a peek on my machine tomorrow, but otherwise will merge in. Only thing is that I think we'd only want to show this if the env variable for the link is populated. Happy to make that change tom.

**Pull request merged**